target/toolchain/files/wrapper.sh

```diff
@@ -43,6 +43,7 @@ TOOLCHAIN_BIN_DIR="$REALNAME_DIR/"
 
 # Set the PATH so that our run-time location is first
 # (get_feature is run from the path, so this has to be set)
+export ORIG_PATH=${ORIG_PATH:-$PATH}
```

Comment:
This looks like it may be redundant? It's just a script that does $stuff without calling $MAKE again?

Reply:
first commit, I'm just putting a variable that stores the original $PATH in every place where it looks like $PATH is changing.

it may not be useful there, but eh why not?

Reply:
for consistency at least...